services/migrations/V1_init_database.ts

```diff
@@ -0,0 +1,50 @@
+import { AddedSiteParams, SiteFeedParams} from "../../constants";
+import { UserFeed } from "../../models";
+import { useCRUD } from "../index";
+import { useDBService } from "../useDBService";
+
+export async function V1_init_database(db: any){
+
+    const { createTable } = useCRUD();
+    const { insertUserFeed } = useDBService();
+
+    await createTable(db, AddedSiteParams);
+    await createTable(db, SiteFeedParams);
+
+    const localStorageData = extractFromLocalStorage();
+    localStorageData.forEach(data => insertUserFeed(data))
+    
+}
+
+interface LocalStorage {
+    items: LocalStorageFeed[];
+    title: string;
+    visited: any
+  }
+
+  interface LocalStorageFeed{
+    title: string;
+    link: string;
+    author: string;
+  }
+
+  function extractFromLocalStorage(): UserFeed[] {
+
+    const userFeedCollection: UserFeed[] = [];
+      
+    for (const key of Object.keys(localStorage)) {
+      if (!key.startsWith("http")) {
+        continue;
+      }
+      const container: LocalStorage = JSON.parse(localStorage.getItem(key) as string);
+
+      var userFeed: UserFeed = {AddedSite: {Url: key, Author: container.items[0].author}, SiteFeed: []}
+
+      container.items.forEach(item => userFeed.SiteFeed?.push({Url: item.link, IsVisited: false, Title: item.title}));
```

Comment:
I don't see a reason why we need to store unvisited URLs.
We should store only visited:
```js
visited: {"https://strdr4605.com/my-first-open-source-project": true,…}
```

Also, why do we need a separate table for visited URLs?
Wouldn't be more performant to store in the same table, as an object and filter as we do with localStorage
```js
// pseudo
storedVisitedForFeed = getVisitedByFeedId(feedId);
const vizitedItems = feed?.items.filter(
            (item) => item.link && storedVisitedForFeed && storedVisitedForFeed[item.link]
          );
```